IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcExcavation/DocEntity.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcExcavation" UniqueId="16107c0c-a0e0-4f60-ad45-4fd438b56863" BaseDefinition="IfcFeatureElementSubtraction" />
```

Comment:
Should it be abstract?

Reply:
I think so since the subclasses are exhaustive as far as I can understand. Will change.